.github/workflows/master.yml

```diff
@@ -182,6 +182,14 @@ jobs:
           java-version: ${{ matrix.java }}
           cache: 'maven'
           check-latest: false
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
```

Comment:
Does pyflink support python 3.9? we use python 3.9 for pyspark, we'd better use the same version if possible.

Reply:
Flink 1.16 only started supporting Python 3.9, so we can't align the Python version to Spark.

Reply:
can you supply a flink/python version matrix in the PR description?

Reply:
I have added version matrix in he PR description.